src/pages/resources/openapi.json

```diff
@@ -9603,6 +9603,23 @@
           "fragments": {
             "description": "Specifies the Fragments JSON object or a JSON array to resolve fragments in the document template. To know more about fragments use-case in document generation and document templates, please see the <a href=\"https://developer.adobe.com/document-services/docs/overview/document-generation-api/fragments/\">documentation</a>.",
             "type": "object"
+          },
+          "notifiers": {
+            "description": "Specifies the array of notifier related information. To know more about notifier related use-case, please see the <a href=\"https://developer.adobe.com/document-services/docs/overview/pdf-services-api/howtos/webhook-notification/\">documentation</a>.",
```

Comment:
Specifies the notifiers to be invoked on the completion of job. To know more about notifiers, please see please see the <a href=\"https://developer.adobe.com/document-services/docs/overview/pdf-services-api/howtos/webhook-notification/\">documentation</a>."